oxapy/__init__.py

```
@@ -285,6 +286,150 @@ def main():
+        field_name: str = "_csrf_token",
+        cookie_max_age: int = 3600,
+        safe_methods: tuple[str, ...] = ("GET", "HEAD", "OPTIONS", "TRACE"),
+    ):
+        self.secret = secret
+        self.cookie_name = cookie_name
+        self.header_name = header_name
+        self.field_name = field_name
+        self.cookie_max_age = cookie_max_age
+        self.safe_methods = safe_methods
+
+    def __call__(self, request, next, **kwargs):
```

Ruff (0.16.1)

[error] Function argument `next` is shadowing a Python builtin

(A002)